docs/source/configuration_file.rst

```diff
@@ -1625,6 +1625,38 @@ Available Transformations:
         }
     }
 
+.. jsonschema::
+
+    {
+        "$schema": "http://json-schema.org/draft-04/schema#",
+        "title": "RandomGamma",
```

Comment:
How about simply calling it `Gamma`?
Usually these transformations are random, and it makes it clearer from a user standpoint (when looking at the doc)

Reply:
I think the explicit usage of "Random" here is helpful because:
* first input param is a range of log gammas, as opposed to a single value of gamma, from where we *randomly* sample a gamma value
* second input param is probability of applying this augmentation, which makes the transformation *random*

It is important to make the distinction between such random augmentations and deterministic augmentations (e.g. `CenterCrop`, `NormalizeInstance`, etc.). Moreover, I see a similar logic applied in other libraries such as [`torchio`](https://torchio.readthedocs.io/transforms/augmentation.html) and [`torchvision`](https://pytorch.org/vision/stable/transforms.html). `ivadomed` also features `RandomAffine`, `RandomShiftIntensity`, and `RandomReverse`.

I am happy to be convinced otherwise, but whatever we decide here, I think we should unify `ivadomed` transformations in terms of this logic. For example, `ElasticTransform` should instead be `RandomElasticTransform` by my logic as it includes ranges and probability as input parameters.

Reply:
you convinced me 😊 

and yes, i suggest the following changes:
- making 'Random' prefix systematic when that applies
- listing all transformatinons alphabetically in the doc for better clarity

Reply:
Opened issue #1030 regarding this.